internal/twirptest/service.twirp.go

```diff
@@ -381,32 +365,23 @@ func writeError(ctx context.Context, resp http.ResponseWriter, err error, hooks
 
 	statusCode := twirp.ServerHTTPStatusFromErrorCode(twerr.Code())
 	ctx = ctxsetters.WithStatusCode(ctx, statusCode)
-	ctx = callError(ctx, hooks, twerr)
+	ctx = callError(ctx, hooks, twerr) // Error hook
```

Comment:
Remove this comment

Reply:
Would it make sense to rename `callError` to `callErrorHook`? and `callResponseSent` to `callResponseSentHook`? etc.  No big deal, I just found "callError" to be a little misleading.